app/views/directives/notifications/notification-body.html

```diff
@@ -0,0 +1,61 @@
+<div
+  uib-dropdown
+  class="dropdown pull-right dropdown-kebab-pf"
+  ng-if="notification.actions && notification.actions.length > 0">
```

Comment:
It would be nice at some point to have actions for certain events, like Build Started -> View Log. Not necessary for this PR, though.